lib/msf/core/auxiliary/udp_scanner.rb

```diff
@@ -97,6 +97,7 @@ def scanner_send(data, ip, port)
     resend_count = 0
     begin
 
+      print_status("Sending packet to #{ip}")
```

Comment:
@wvu-r7 looks like many modules using `UDPScanner` print their own message before sending, Example: modules/auxiliary/scanner/ipmi/ipmi_cipher_zero.rb so maybe make this one a `vprint_status`?